### DISCUSSION TITLE
Acquisition Values negative with qLogEI

### DISCUSSION THREAD

**Discussion by mvincent-ethzBA** [meta-pytorch/botorch#2534] [Q&A] [answered]
Hello everyone,

After changing my acquisiton function from ```qExpectedImprovement``` to ```qLogExpectedImprovement```, my aquisition value is suddenly negative. I have looked over the code and I haven't been able to find anything that I might have coded wrong. 
However, I have noticed that the negative acquisition value, if used as the exponent with 10 as the base (```10^acq_value```), is roughly in the general expected area when that expression is evaluated. I could, however, not find any documentation that would answer this question which is why I'm asking here.

Is this behaviour expected, and the negative acquisition value is indeed just the extracted exponent after taking the decimal logarithm, or did something go wrong in the calculation and I need to look over my code again?

**Comment by Balandat** [comment, marked as answer]
This is expected - `log(x) < 0` for `x < 1` and EI will often be less than one (that's an oversimplification, logEI is not just taking the log of EI but computes the value in a numerically robust fashion, but that's the gist of it). Since `log` is monotonic the maximizer of logEI is the maximizer of EI.

**Reply by mvincent-ethzBA** [reply]
Good to have it confirmed. Thanks for your quick answer!
